src/Admin/Model/UserAdmin.php

```diff
@@ -251,4 +251,16 @@ protected function configureFormFields(FormMapper $formMapper): void
             ->end()
         ;
     }
+
+    private function isAllowedToSwitch(): bool
+    {
+        $role = 'ROLE_ALLOWED_TO_SWITCH';
+        $firewallMap = $this->getConfigurationPool()->getContainer()->get('security.firewall.map');
```

Comment:
Since this configuration is [provided by "symfony/security-bundle"](https://github.com/symfony/security-bundle/blob/97ba8648e718999793e79ab4d1f1582c8d19be9d/DependencyInjection/MainConfiguration.php#L264-L275), I think we should check for the availability of this 
 configuration before, or add the requirement explicitly.

Reply:
I add `->get('security.firewall.map', ContainerInterface::NULL_ON_INVALID_REFERENCE)` and check if firewall.map is not null.
Is it good ?